src/components/issue-list.jsx

```diff
@@ -122,11 +122,17 @@ const IssueList = React.createClass({
 
     const header = (
       <h2 className={className}>
-        {iconEl}
-        {title}
-        {' ('}
-        {countOrDownloadLink}
-        {')'}
+        {iconEl}{title} ({countOrDownloadLink})
+        {primaryRepo && <a className="add-issue"
+          href={
+            'https://github.com/' + primaryRepo.repoOwner + '/' + primaryRepo.repoName +
```

Comment:
Nice! I'm not sure how well this would work with users of GitHub Enterprise (see #23). You _should_ be able to use `window.localStorage.setItem('gh-rootURL', rootURL)` and maybe would have to remove the trailing `/api/v3` or just change the code in #23 to not ask for the `/api/v3` at the end (probably cleaner)